ncm-nrpe/src/main/perl/nrpe.pod

```diff
@@ -11,80 +11,84 @@ way.
 
 =head1 COMPONENT STRUCTURE
 
-All fields are mandatory, unless it is explicitly stated:
+All fields are required (but most have sensible defaults) unless otherwise stated:
```

Comment:
should be `All fields are required (but most have sensible defaults unless otherwise stated)`? (because you add phrases like `Must be specified, no default is provided.`).